### ISSUE TITLE
Remove plain password input into environmental vars

### ISSUE THREAD

**Issue by harleylang** [harleylang/hydroxide-docker#1]
See: https://github.com/emersion/hydroxide/issues/109#issuecomment-658490964

**Comment by arichtman** [comment]
I note that comment linked is around the TLS wrapper security - I can add that by adding a Traefik service into the Docker-Compose.
However the _title_ for this issue is talking about plain password input - I can see a few issues around security - shall I just add them myself seperately?

**Comment by harleylang** [comment]
@arichtman yes any PRs that seeks to tighten the security of the Dockerfile / docker-compose (including best practices for inputting / storing passwords) are welcome!!!
